cmd/olympus/actions/app.go

```diff
@@ -3,19 +3,29 @@ package actions
 import (
 	"log"
 
+	"github.com/garyburd/redigo/redis"
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/middleware"
 	"github.com/gobuffalo/buffalo/middleware/ssl"
+	"github.com/gobuffalo/buffalo/worker"
 	"github.com/gobuffalo/envy"
 	"github.com/rs/cors"
 	"github.com/unrolled/secure"
 
 	"github.com/gobuffalo/buffalo/middleware/csrf"
 	"github.com/gobuffalo/buffalo/middleware/i18n"
+	"github.com/gobuffalo/gocraft-work-adapter"
 	"github.com/gobuffalo/packr"
 	"github.com/gomods/athens/pkg/cdn/metadata/azurecdn"
 )
 
+const (
+	// PushWorkerName is the name of the worker processing pushes
+	PushWorkerName            = "pushProcessor"
```

Comment:
change this to olympusWorker, you will have some conflicts as i already added one worker
if you're in doubt follow pattern in athens 1worker-2 handlers

Reply:
Aren't there two workers in athens @michalpristas ?
https://github.com/gomods/athens/blob/c4ac27f58527e3cfc170cb0393f89f9a5a0df202/cmd/proxy/actions/app.go#L23-L26